spec/authorization_spec.rb

```diff
@@ -59,4 +59,80 @@
       end.should raise_exception(RestClient::Unauthorized)
     end
 
+  it "super admin endpoints should reject activation key auth" do
+    no_auth_client = Candlepin.new
+    @cp.create_activation_key(@owner['key'], 'key1')
+
+    lambda do
+      no_auth_client.create_owner_with_activation_key(random_string("test_owner"), @owner['key'], ['key1'])
+    end.should raise_exception(RestClient::Forbidden)
+  end
+
+  it "verified endpoints should reject activation key auth" do
+    no_auth_client = Candlepin.new
+    @cp.create_activation_key(@owner['key'], 'key1')
+
+    lambda do
+      no_auth_client.get_owner_with_activation_key(@owner['key'], ['key1'])
+    end.should raise_exception(RestClient::Forbidden)
+  end
+
+  it "default security hole should reject activation key auth" do
+    no_auth_client = Candlepin.new
+    @cp.create_activation_key(@owner['key'], 'key1')
+
+    lambda do
+      owner = no_auth_client.get_product_with_activation_key(@owner['key'], 'test_prod', ['key1'])
```

Comment:
The name of the test says it's exercising a default security hole endpoint, but this method isn't actually hitting an endpoint with SecurityHole

Reply:
Done. I was calling the wrong endpoint.